src/components/TodoComponent/TodoComponent.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todo: Todo;
+  onRemove: (todoData: Todo) => void;
+}
+
+export const TodoComponent: FC<Props> = React.memo(({
+  todo,
+  onRemove,
+}) => {
+  const { title, completed } = todo;
+
+  const handleDeleteTodo = () => {
+    onRemove(todo);
+  };
+
+  return (
+    <div className={classNames('todo', { completed })}>
+      <label className="todo__status-label">
+        <input
+          type="checkbox"
+          className="todo__status"
+          checked={completed}
+        />
+      </label>
+
+      <span
+        className="todo__title"
+      >
```

Comment:
single small properties can lie in one line
```suggestion
      <span className="todo__title">
```